src/store/modules/user/actions.ts

```diff
@@ -8,39 +8,45 @@ import { translate } from '@/i18n'
 import { Settings } from 'luxon'
 import { getServerPermissionsFromRules, prepareAppPermissions, resetPermissions, setPermissions } from '@/authorization'
 import { updateInstanceUrl, updateToken, resetConfig } from '@/adapter'
-
 import logger from "@/logger";
 
 const actions: ActionTree<UserState, RootState> = {
 
   /**
    *  Login user
    * @param param0 state context
-   * @param param1 payload: object { username, password }
+   * @param param1 payload: token and oms
    * @returns Promise
    */
-  async login({ commit }, { username, password }) {
-    try {
-      const resp = await UserService.login(username, password);
-      // Further we will have only response having 2xx status
-      // https://axios-http.com/docs/handling_errors
-      // We haven't customized validateStatus method and default behaviour is for all status other than 2xx
-      // TODO Check if we need to handle all 2xx status other than 200
+  // async login({ commit }, { username, password }) {
+  //   try {
+  //     const resp = await UserService.login(username, password);
+  //     // Further we will have only response having 2xx status
+  //     // https://axios-http.com/docs/handling_errors
+  //     // We haven't customized validateStatus method and default behaviour is for all status other than 2xx
+  //     // TODO Check if we need to handle all 2xx status other than 200
 
 
-      /* ---- Guard clauses starts here --- */
-      // Know about Guard clauses here: https://learningactors.com/javascript-guard-clauses-how-you-can-refactor-conditional-logic/
-      // https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a
+  //     /* ---- Guard clauses starts here --- */
+  //     // Know about Guard clauses here: https://learningactors.com/javascript-guard-clauses-how-you-can-refactor-conditional-logic/
+  //     // https://medium.com/@scadge/if-statements-design-guard-clauses-might-be-all-you-need-67219a1a981a
 
 
-      // If we have any error most possible reason is incorrect credentials.
-      if (hasError(resp)) {
-        showToast(translate('Sorry, your username or password is incorrect. Please try again.'));
-        logger.error("error", resp.data._ERROR_MESSAGE_);
-        return Promise.reject(new Error(resp.data._ERROR_MESSAGE_));
-      }
+  //     // If we have any error most possible reason is incorrect credentials.
+  //     if (hasError(resp)) {
+  //       showToast(translate('Sorry, your username or password is incorrect. Please try again.'));
+  //       logger.error("error", resp.data._ERROR_MESSAGE_);
+  //       return Promise.reject(new Error(resp.data._ERROR_MESSAGE_));
+  //     }
 
-      const token = resp.data.token;
+  //     const token = resp.data.token;
+  async getAndSetUserDetails({ commit, dispatch, state }, payload) {
+    try {
+      const { token, oms } = payload
+      const baseURL = process.env.VUE_APP_BASE_URL
```

Comment:
This will be part of login plugin only

Reply:
Removed sir, this is handled in the Login plugin in the Launchpad app.